cmd/appliance/functions/download.go

```diff
@@ -139,6 +141,26 @@ func NewApplianceFunctionsDownloadCmd(f *factory.Factory) *cobra.Command {
 				opts.version = tag
 			}
 
+			opts.destination = filesystem.AbsolutePath(opts.destination)
+			if _, err := os.Stat(opts.destination); err != nil {
+				createDir := true
+				if f.CanPrompt() {
+					if err := prompt.SurveyAskOne(&survey.Confirm{
+						Message: fmt.Sprintf("Directory '%s' does not exist. Do you want to create it now?", opts.destination),
+					}, &createDir); err != nil {
+						return err
+					}
+				} else {
+					fmt.Fprintf(opts.out, "Directory '%s' does not exist\n", opts.destination)
+				}
+				if createDir {
```

Comment:
createDir always true?

Reply:
no, not if the user says 'no' in the prompt. Setting it to true just makes it the default.

Reply:
ah.. its used as a reference there. i did not see that